api/prisma/migrations/migrate.lock

```diff
@@ -1,6 +1,4 @@
-# IF THERE'S A GIT CONFLICT IN THIS FILE, DON'T SOLVE IT MANUALLY!
-# INSTEAD EXECUTE `prisma migrate fix`
 # Prisma Migrate lockfile v1
-# Read more about conflict resolution here: TODO
 
-20200519151236-create-schema
\ No newline at end of file
+20200519151236-create-schema
+20200819173245-migration
```

Comment:
Can you just delete the existing migration completely and redo so that we have a single one, "create-schema"

Reply:
makes sense. doing.